docs/usage/connections.md

```diff
@@ -2,19 +2,19 @@
 
 In order to build a useful application, Atmo needs to be able to connect to external resources. Currently, Atmo can connect to [NATS](https://nats.io/) and [Redis](https://redis.io/), and upcoming releases will include additional types such as databases and more.
 
-To create connections, add a `connections` section to your Directive:
+To create connections, add a `connections` section to your Directive. When Atmo starts up, it will establish the connections you've configured, and make them available to your application in a few different ways.
```

Comment:
```suggestion
To create connections, add a `connections` section to your Directive. When Atmo starts up, it will establish the connections you've configured, and make them available to your Runnables.
```

Reply:
It's not just for the Runnables, they can also be used as stream sources for handlers.